MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -231,10 +229,111 @@ Change the default colors of the `ColorPalette` in the toolbar.
+saveItem.IsEnabled = false;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Customize the Toolbar Item View
+
+To customize the view of toolbar items, specify their icon names and set the view using the `View` property in the `ImageEditorToolbarItem`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem)headerToolbar.ToolbarItems[2];
+ImageEditorToolbarItem saveItem = saveGroup.Items.FirstOrDefault(i => i.Name == "Save");
+
+Image image = new Image();
+image.Source = ImageSource.FromFile("save.jpg");
+saveItem.View = image;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Add the SubToolbar
+
+Create `SubToolbars` within your toolbar items to organize related actions.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+ImageEditorToolbar footerToolbar = imageEditor.Toolbars[1];
+ImageEditorToolbarItem cropItem = (ImageEditorToolbarItem)footerToolbar.ToolbarItems.FirstOrDefault(i => i.Name == "Crop");
+cropItem.SubToolbarOverlay = false;
+cropItem.SubToolbars = new List<ImageEditorToolbar>()
+{
+    new ImageEditorToolbar()
+    {
+        ToolbarItems = new List<IImageEditorToolbarItem>()
+        {
+           new ImageEditorToolbarItem(){ Name = "original"},
+           new ImageEditorToolbarItem(){ Name = "circle"},
+           new ImageEditorToolbarItem(){ Name = "square"}
+        },
+    }
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Add a Toolbar Item with the Existing Toolbar
+
+Explicitly add the `ImageEditorToolbarItem` to the `ImageEditorToolbar` manually.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+Image image = new Image();
+image.Source = ImageSource.FromResource("MyProject.Resources.Images.share.png");
+image.WidthRequest = 48;
+image.HeightRequest = 48;
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem)headerToolbar.ToolbarItems[2];
+saveGroup.Items.Add(new ImageEditorToolbarItem() { Name = "Share", View = image });
+
+{% endhighlight %}
+
+{% endtabs %}
+
 ## Explicitly Create Toolbars
 
 The ImageEditor automatically generates the toolbars. To add the toolbars manually, you need to explicitly add them and change the `AutoGenerateToolbarItems` property to `false`.
 
+N> When you create a new toolbar item, you can use pre-existing built-in names for the item, which will automatically generate the icon view. If you want to introduce new icons or modify the current one, you can use the View property of the `ImageEditorToolbarItem` class to define the item view.
```

Comment:
When you create a new toolbar item, you can use the pre-existing built-in names for the item, which will automatically generate the icon view. If you want to introduce a new item view or modify the current one, you can use the `View` property of the `ImageEditorToolbarItem` class to define the item view.

Reply:
updated.